stable/admin/README.md

```diff
@@ -283,7 +254,7 @@ Verify the pods are running:
 ```bash
 $ kubectl get pods
 NAME                           READY     STATUS    RESTARTS   AGE
-admin-nuodb-0                         1/1       Running   0          29m
+admin-nuodb-0                  1/1       Running   0          29m
```

Comment:
The pod name is different nowadays. We added cluster id to the name.

Reply:
I am not going to fix all the docs with this review. There are many things that need to be changed.